index.html

```diff
@@ -0,0 +1,137 @@
+                    consectetur. Suspendisse rhoncus scelerisque sem, ut elementum justo dignissim in. Nulla ut posuere
+                    orci. Curabitur eleifend, erat blandit rhoncus molestie, odio lacus gravida sem, at finibus tortor
+                    lectus quis lacus. In eget porttitor est, ac feugiat nisl.</p>
+                <p>Nam faucibus nibh nunc, at dictum mauris blandit vitae. In eleifend mollis ipsum et commodo. Sed
+                    justo urna, sagittis et semper quis, suscipit vel lacus. Aliquam orci diam, pulvinar mattis tellus
+                    nec, mollis feugiat erat. Nam suscipit, ex sit amet pellentesque vehicula, metus dolor faucibus est,
+                    sed condimentum velit lorem sed nulla. Curabitur nibh nisi, vestibulum sit amet magna at, pharetra
+                    pellentesque nisl. Aenean dictum, nibh eu congue elementum, libero mauris laoreet mauris, id
+                    porttitor lorem arcu accumsan justo. Nullam molestie, libero in interdum viverra, felis orci auctor
+                    elit, eu sollicitudin lacus enim ut dolor. Etiam non nunc ac nulla facilisis sagittis quis sit amet
+                    odio. Suspendisse sit amet lacinia ex. Phasellus consequat pulvinar nibh ut ultrices. Vestibulum
+                    faucibus faucibus orci a imperdiet. Nulla facilisi. Donec eget massa eget sapien semper dignissim
+                    posuere et leo.</p>
+                <code>
+                let nickName = "Stany"
+              </code>
+            </section>
+            <section id="Topic_Five" class="main-section">
+                <header>
+                    <h1> Topic Five </h1>
+                </header>
+                <p>It's very fine for me</p>
+                <p>May God bless you!</p>
+                <p>I'm very happy, thanks Microverse for your oportunity and thanks Andres for your time and your
+                    patience! </p>
+                <p>Donec egestas lacus id velit elementum, nec porta magna auctor. Etiam dignissim congue blandit.
+                    Nullam ex nulla, fringilla a faucibus non, mattis tincidunt eros. In volutpat purus non lacus
+                    scelerisque, iaculis suscipit augue vehicula. Nam dapibus convallis ex, et pulvinar diam tincidunt
+                    ac. Nulla vehicula lacus ut lorem mollis, eget gravida ex lobortis. Aliquam gravida leo bibendum
+                    metus eleifend, quis condimentum neque porttitor. Integer feugiat leo orci, eget laoreet ex
+                    tincidunt vitae. Sed ullamcorper aliquam iaculis. Aenean id metus enim. In a tellus eu justo blandit
+                    venenatis. Nullam tristique felis at tincidunt fringilla. Proin suscipit suscipit nulla, in blandit
+                    elit mollis ut. Donec porttitor euismod dolor, malesuada ornare velit dictum quis. </p>
+                <code>
+                let countryName = "Haiti"
+              </code>
+            </section>
+
```

Comment:
Remove no needed space

Reply:
- [ ] I think your comment should be more polite. So it should also be a suggestion instead of making it look like an order.  For example, you can say, "**_Kindly_** try to remove white spaces in your code. They will make your project look messy"